src/universal/components/RetroSidebarDiscussSection.js

```diff
@@ -67,16 +44,30 @@ const RetroSidebarDiscussSection = (props: Props) => {
         const {reflectionGroup} = stage;
         if (!reflectionGroup) return null;
         const {title, voteCount} = reflectionGroup;
+        // the local user is at another stage than the facilitator stage
+        const isOutOfSync = !inSync && stage.id === facilitatorStageId;
```

Comment:
i'm not a fan of the variable name, but i see why it's hard to find a good name. maybe `isUnsyncedFacilitatorStage`? Usually when we talk about inSync, we are referring to the viewer, here we're referring to the stage, so it's a bit of a gotcha for comprehension.